datacenter/dtr/2.4/guides/index.md

```diff
@@ -34,10 +34,10 @@ and metadata such that if a machine fails, DTR continues to operate and can be r
 
 ## Efficiency
 
-DTR has the ability to [cache images closer to users](../admin/configure/deploy-caches/index.md)
+DTR has the ability to [cache images closer to users](admin/configure/deploy-caches/index.md)
```

Comment:
@joaofnfernandes , is this link going  to work? It doesn't for me.

Reply:
This works: `admin/configure/deploy-caches/`

Reply:
Both urls should be valid. I've tested the one ending in `.md` with netlify and and local build and it seems to work. Maybe I'm missing something 🤔

Reply:
Probably not. Go for it.